xbmc/URL.cpp

```diff
@@ -494,6 +488,16 @@ const std::string CURL::GetFileNameWithoutPath() const
   return URIUtils::GetFileName(file);
 }
 
+inline
+void protectIPv6(std::string &hn)
```

Comment:
Not into the details but is this bullet proof for reliably detecting an ipv6 hostname? Just double checking - no doubt you know what you are doing here ;) this basically checks if there is more then one colon in the hostname - right?